replication-api/src/main/java/com/connexta/replication/api/data/Persistable.java

```diff
@@ -22,18 +22,4 @@
    * @return The ID of the object
    */
   String getId();
```

Comment:
❓ Sometimes is it useful to track the last time a configuration was modified and the user who modified it. Does anyone see any value in adding these 2 fields? I don't know if we would add them here or on an as needed basis per configuration.

Reply:
I agree that these are a very good thing to have for traceability. 

I think we can wait before adding then since we currently do not have a UI or a way to create/update these objects from a user. As such, it might provide little value (aside maybe for the last modified timestamp).